tracee-ebpf/tracee/tracee.go

```diff
@@ -736,6 +736,24 @@ func (t *Tracee) populateBPFMaps() error {
 	bpfConfigMap.Update(uint32(configFollowFilter), boolToUInt32(t.config.Filter.Follow))
 	bpfConfigMap.Update(uint32(configDebugNet), boolToUInt32(t.config.Debug))
 
+	// Initialize pid_to_cont_id_map if tracing containers
+	if t.config.Filter.ContFilter.Value {
+		bpfPidToContIdMap, _ := t.bpfModule.GetMap("pid_to_cont_id_map")
+		c := Containers{}
+		err := c.Populate()
+		if err != nil {
+			return fmt.Errorf("failed to populate pid_to_cont_id_map")
+		}
+		for fullcontid, pidarray := range c.Get() {
+			for _, pidstr := range pidarray {
+				container_id := fullcontid[:12]
```

Comment:
I thought all dirs from inside a 'docker' directory within a cgroup fs would have obligatory the same amount of chars for the container_id (and was substring-ing-it to 12 because of the eBPF map). I do see your point in checking if there are 'other dirs' inside cgroups docker directory.